content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -22,11 +22,21 @@ To install Kiali without using Helm, following the [Kiali install instructions](
 Create a secret in your Istio namespace with the credentials that you use to
 authenticate to Kiali. See the
```

Comment:
We should not cross-reference the Helm readme.  Can you address this?  I'd like to remove the helm readme entirely.  Unfortunately  Helm presents the readme in various product UIs.  Unfortunately this requires documentation in two places which is evil.  But it is what it is.

Cheers
-steve

Reply:
> We should not cross-reference the Helm readme. Can you address this?

This is done - see this PR's commit d4e0714de77f523e2b8b7b864567186b95a8b9d9